core/bones/relational.py

```diff
@@ -20,6 +21,12 @@ class RelationalConsistency(Enum):
     CascadeDeletion = 4  # Delete this object also if the referenced entry is deleted (Dangerous!)
 
 
+class RelationalUpdateLevel(Enum):
+    Allways = 0
+    RebuildSearchIndex=1
+    IfSet=2
```

Comment:
Please use PEP-8 style
```suggestion
    RebuildSearchIndex = 1
    IfSet = 2
```